packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -220,6 +228,7 @@ class FlutterCommandRunner extends CommandRunner {
     // Set up the tooling configuration.
     if (enginePath != null) {
       ToolConfiguration.instance.engineSrcPath = enginePath;
```

Comment:
This will be centralized in ToolConfiguration.  enginePath is obsolete and will be removed along with BuildConfigurations.